Automattic/newspack-listings
# chore: replace AutocompleteTokenfield with Newspack Components version

### All Submissions:






### Changes proposed in this Pull Request:



Replaces the custom `AutocompleteTokenfield` component with the one from Newspack Components. Still requires a few CSS overrides, so leaving the SCSS file in place with some modifications.

Also opens up potential usage of other Newspack Components.

Closes #4.

### How to test the changes in this Pull Request:

1. Check out this branch, run `npm ci && npm run build` to build with the Newspack Components package.
2. Create a Curated List block in query mode. Test all autocomplete fields under Query Settings in the sidebar and confirm that all styles and behavior are as expected.
3. Repeat with a Curated List block in Specific Listings mode: test the search field and confirm that all styles and behavior are as expected.

### Other information:
